vnext/Microsoft.ReactNative/Views/TouchEventHandler.cpp

```diff
@@ -449,49 +469,76 @@ bool TouchEventHandler::TagFromOriginalSource(
   return true;
 }
 
-winrt::IPropertyValue TouchEventHandler::TestHit(
-    const winrt::Collections::IVectorView<xaml::Documents::Inline> &inlines,
-    const winrt::Point &pointerPos,
-    bool &isHit) {
-  winrt::IPropertyValue tag(nullptr);
-
-  for (const auto &el : inlines) {
-    if (const auto span = el.try_as<xaml::Documents::Span>()) {
-      auto resTag = TestHit(span.Inlines().GetView(), pointerPos, isHit);
+bool IsCharacterBefore(winrt::TextPointer textPointer, winrt::Point point) {
+  const auto rect = textPointer.GetCharacterRect(winrt::LogicalDirection::Forward);
+  const auto bottom = rect.Y + rect.Height;
+
+  // The character Rect always has Width = 0, so we need to use the X-dimension
+  // of the next character on the same line. If the next character is not on
```

Comment:
this could give some _slightly wrong_ calculations for ligatures 
![image](https://user-images.githubusercontent.com/22989529/114067115-4e65cf00-9851-11eb-8240-4b69770731e9.png)

It might not be a big deal but thought I'd mention it anyway

Reply:
Ligatures should not be an issue. We only need the hit testing to work at the granularity of the Inline (run) and I doubt a ligature would cross the boundary of an Inline (please correct me if I'm wrong).